ctwrap/modules/adiabatic_flame.py

```diff
@@ -52,14 +50,12 @@ def run(name,
         loglevel   (int): amount of diagnostic output (0 to 8)
 
     Returns:
-        dict: dictionary with pandas DataFrame entries
+        Out (dict): dictionary containing cantera Flamebase object
```

Comment:
same as above ... merge `Arguments`/`Keyword Arguments` 

Also, skip `Out (dict):` for return ... it should just be
```
Returns:
    Dictionary containing Cantera `FlameBase` object.
```